lib/rubocop/cop/mixin/comments_help.rb

```diff
@@ -75,6 +77,14 @@ def find_end_line(node)
         end
       end
       # rubocop:enable Metrics/AbcSize
```

Comment:
Following the comment above.

```suggestion
      # rubocop:enable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```